gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/DataFlowRoutesPicker.java

```diff
@@ -0,0 +1,14 @@
+package gobblin.data.management.copy.replication;
+
+import com.typesafe.config.Config;
+
+
+/**
+ * Used to pick preferred {@link DataFlowTopology.CopyRoute} in {@link com.typesafe.config.Config} format
+ * @author mitu
+ *
+ */
+public interface DataFlowRoutesPicker {
```

Comment:
I still don't understand. How do you instantiate a DataFlowRoutesPicker implementation?